src/main/resources/templates/python/test/structural/structural_test.py

```diff
@@ -1,8 +1,8 @@
 import unittest
-from assignment import sorting_algorithms
-from assignment import sort_strategy
-from assignment import context
-from assignment import policy
+from ${studentParentWorkingDirectoryName} import sorting_algorithms
+from ${studentParentWorkingDirectoryName} import sort_strategy
+from ${studentParentWorkingDirectoryName} import context
+from ${studentParentWorkingDirectoryName} import policy
```

Comment:
I don't fully get how this works here, but I guess I missed the part where the files (in this case .py) are transformed to a correct file. Where does this happen?

Reply:
This will be replace during the exercise creation. Python has a special case where the path (e.g. assignment/code) has to be change to (assignment.code). This happens in [this file](https://github.com/ls1intum/Artemis/pull/9217/files#diff-20411055de836e298f72ea8eaa62b094edeeae51b7073d159b9ef9572d819b9f)